src/game/world/map/Tiles.cpp

```diff
@@ -317,7 +318,9 @@ void Tiles::RaiseAllTilesBy( Tile::elevation_t amount ) {
 			tiles.push_back( At( x, y ) );
 		}
 	}
-	random_shuffle( tiles.begin(), tiles.end() );
+	std::random_device rd;
+	std::mt19937 g(rd());
+	std::shuffle( tiles.begin(), tiles.end(), g);
```

Comment:
random_shuffle is deprecated, this is the new way to do it
https://en.cppreference.com/w/cpp/algorithm/random_shuffle